docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+---
+title: Pods
+---
+
+# Pods
+
+Marathon supports the creation and management of pods. Pods enable you to share storage, networking, and other resources among a group of applications on a single agent and address them as one group rather than as separate applications, similar to a virtual machine. Pods allow quick, convenient coordination between applications that need to work together, for instance a database and web server that make up a content management system.
```

Comment:
Agree, should not compare this to a VM. We should reference specific common sidecar use cases like caching, metrics, etc.